docs/http/traffic-policy/actions/redirect.mdx

```diff
@@ -0,0 +1,38 @@
+import { ExampleHTTP } from "/examples/actions/redirect.mdx";
+
+# Redirect
+
+## Overview
+
+The Redirect policy action enables transforming your URL using regular expressions and redirect through location header.
+
+## Example
+
+<ExampleHTTP />
+
+## Behavior
+
+When executed this action will abort the request and replace all matches on the inbound URL with the configured replacements. Before regular expression replacement occurs, all named variables will be replaced with their corresponding values.
+
+<VariableInterpolation type="redirect" />
+### Variables
+<HttpHeaderTemplates />
+
+## Reference
+
+### Supported Directions
+
+- Inbound
+
+### Configuration
+
+| Type       |
+| ---------- |
+| `redirect` |
+
+| Parameter     | &nbsp;                    | Description                                                                                                                                       |
+| ------------- | ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `from`        | string                    | A regular expression string to match inside of the URL.                                                                                           |
+| `to`          | string                    | A regular expression string to replace the `from` match with. Also contains special [named variables](#named-variables) outside of regular regex. |
+| `status_code` | int                       | 3xx status code used for redirecting, 302 by default.                                                                                             |
+| `headers`     | Map&lt;string, string&gt; | Headers to be added to the request.                                                                                                               |
```

Comment:
should I add named variables?

Reply:
oooo yes I think so — it's also the same stuff reused right?